uplink/storage/streams/store.go

```diff
@@ -462,11 +463,13 @@ func pathForKey(raw string) paths.Unencrypted {
 func (s *streamStore) List(ctx context.Context, prefix Path, startAfter, endBefore string, pathCipher storj.CipherSuite, recursive bool, limit int, metaFlags uint32) (items []ListItem, more bool, err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	if metaFlags&meta.Size != 0 {
-		// Calculating the stream's size require also the user-defined metadata,
-		// where stream store keeps info about the number of segments and their size.
-		metaFlags |= meta.UserDefined
-	}
+	// TODO use flags with listing
+
+	// if metaFlags&meta.Size != 0 {
```

Comment:
Is this code a companion of the previous TODO?

Reply:
this is something what most probably will be fixed in next PRs. TODO is to rest of method body below.

Reply:
I was not fully clear because there is a blank line between the 2

Reply:
ok, I will change that